Apps/WebClient/src/ClientApp/src/assets/scss/bcgov/_overrides.scss

```diff
@@ -78,3 +78,8 @@ footer nav {
 .alert.alert-warning {
   border-color: theme-color-level(warning, -4);
 }
+
+// Fixes datepicker displaing under site header on mobile
+.dropdown-menu {
+  z-index: 9999;
```

Comment:
is this absolutely necessary? it looks hacky. It would be better if set the Z level on the variables and see what should be on top of what

Reply:
It fixes the issue commented above

Reply:
and yes it is a hack

Reply:
right, but it is kind of heavy handed compared to our other z-index changes. If the class can be put then that would be better.

Reply:
I didn't get your suggestion, what do you mean?

Reply:
Fixed